.ci/magician/cmd/generate_comment.go

```diff
@@ -685,6 +685,9 @@ func init() {
 func checkDocumentFrontmatter(repo source.Repo) []string {
 	var errs []string
 	for _, f := range repo.ChangedFiles {
+		if !strings.HasPrefix(f, "website/docs/r/") && !strings.HasPrefix(f, "website/docs/d/") {
```

Comment:
Given we're removing the subcategory check, I think we could have the frontmatter check across all doc files in website/docs